Felienne/hedy
# Adds keywords translations dropdown to code page UI

**Description**
In this PR we add a new dropdown element to the code page UI to make the translatable keywords more visible. The dropdown menu contains the English keywords as well as the translated keywords (if they exist). In the case the user selects English or a non-keyword language the dropdown is hidden.

**Fixes**
This PR addressed (but does not fully fix) #3128

**How to test**
Navigate to the code page and notice the new dropdown menu. Select both a supported and non-supported keyword language. Verify that the dropdown is hidden when no keywords are translated. Verify that this works for multiple languages and levels. Also verify RTL support by selecting Arabic (or any there RTL language).
